api/v1beta1/verticadb_types.go

```diff
@@ -176,6 +176,10 @@ type VerticaDBSpec struct {
 	// operator adds the same volume mounts that are in the vertica server
 	// container to each sidecar container.
 	Sidecars []corev1.Container `json:"sidecars,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// Custom volumes that can be added to sidecars.
```

Comment:
Lets expand on this comment a bit since it will appear in our CRD.  How about this:
```
// Custom volumes that are added to sidecars.  Each sidecar must include
// the volume in the volumeMounts for it to appear in the container.  It
// accepts any valid volume type.  A unique name must be given for each
// volume and it cannot conflict with any of the internally generated volumes.
```

Reply:
I was lacking inspiration on that part.